Branch-SDK/src/main/java/io/branch/referral/BranchUtil.java

```diff
@@ -47,6 +48,11 @@ static void shutDown() {
      * false if "io.branch.sdk.TestMode" is not added in the manifest or String res.
      */
     static boolean checkTestMode(Context context) {
+        BranchJsonConfig jsonConfig = new BranchJsonConfig(context);
+        if (jsonConfig.isValid()) {
+            isTestModeEnabled_ = jsonConfig.getUseTestInstance();
```

Comment:
Should `isManifestTestModeEnabled` be assigned the `testMode` value too? Just like line 70?